pages.de/windows/robocopy.md

```diff
@@ -0,0 +1,33 @@
+# robocopy
+
+> Robustes Kopieren von Dateien und Ordnern.
+> Standardmäßig werden Dateien nur kopiert, wenn die Quell- und Zieldatei ein unterschiedliches Änderungsdatum oder eine unterschiedliche Dateigröße haben.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/windows-server/administration/windows-commands/robocopy>.
```

Comment:
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/robocopy>.
```
We don't specify l10n in Microsoft URLs from what I know the page automatically redirects you to your native page based on your language setting.

Reply:
Just checked and you are right on that. Fixed the URL With your suggestion.